omod/src/main/java/org/openmrs/web/controller/user/UserFormController.java

```diff
@@ -142,7 +151,8 @@ public String handleSubmission(WebRequest request, HttpSession httpSession, Mode
 	        @RequestParam(required = false, value = "confirm") String confirm,
 	        @RequestParam(required = false, value = "forcePassword") Boolean forcePassword,
 	        @RequestParam(required = false, value = "roleStrings") String[] roles,
-	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson,
+	        @RequestParam(required = false, value = "createNewPerson") String createNewPerson, 
+	        @RequestParam(required = false, value = "providerCheckBox") String addToProviderTableOption,
```

Comment:
And what I'm trying to saying is that in future it's good practice to use a boolean value for a checkbox instead of a String in the HTML like you did in the JSP, that way the type conversion to boolean is automatic on the server side